packages/core/src/components/Selectors/Selectors.styled.tsx

```diff
@@ -30,7 +30,7 @@ export const SelectorLabel = styled(Text)<SelectorLabelProps>`
     ${({ required }) => required && asterisk}
     cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'inherit')};
     color: ${({ disabled, theme, type }) => theme[type].labelColor[disabled ? 'disabled' : 'default']};
-    padding: ${({ labelPosition }) => labelPosition && (labelPosition === 'top' || labelPosition === 'bottom' ? '1.6rem 0' : '0 1.6rem')};
+    padding: ${({ labelPosition, theme, type }) => labelPosition && ( labelPosition === 'top' || labelPosition === 'bottom' ? '1.6rem 0' : type === 'checkbox' ? theme.checkbox.labelPadding : theme.radio.labelPadding )};
```

Comment:
We have to think for padding when `labelPosition === 'top' || labelPosition === 'bottom'`

Reply:
Also we are not passing `type` from `Radio` & `Checkbox` component and hence all the snapshots are failing.

Reply:
@gmukul01 should we include a custom top/bottom padding for the label as well?

Reply:
Another way we can implement the custom label padding is by adding `labelPadding` to the `Checkbox`, `Radio` and `SelectorLabel` FC props. That prop can be optional and used for `padding` if a value exists. Idea below:
```suggestion
   padding: ${({ labelPadding, labelPosition }) => labelPadding || ( labelPosition && (labelPosition === 'top' || labelPosition === 'bottom' ? '1.6rem 0' : '0 1.6rem'))};
```

Reply:
I guess we have to add proper padding in both radio and checkbox like this `labelPadding: {topPositioned: string; bottomPositioned: string; leftPositioned: string; rightPositioned;}`. But still let's discuss over call.

Reply:
Based on the logic in this line, I think either of these approaches will work as a way to provide a dynamic value based on label position but I think it may make sense to keep this value in the theme if possible like Mukul has suggested because it is just a CSS value. 

The prop option also works, but I think we typically try to avoid using props for styles which can be directly translated into a CSS property because there is not really any logic involved for the component (the component will just be passing the value along to be used in the Styled Component). Also, if this value is passed as a prop, the value will have to be redefined in any place this component is used.

Reply:
@gmukul01 and @jaredgiangrasso thank you for your feedback and suggestions! I've made some updates: adjusted both `Radio` and `Checkbox` themes to include a `labelPadding` object, similar to what @gmukul01 suggested. The default values are set to the original padding values for `top/bottom/left/right`. Plmk your thoughts!